.github/workflows/_publish-internal.yml

```diff
@@ -6,11 +6,7 @@ on:
             package:
                 description: "Choose the package to publish"
                 type: string
-                default: "dbt-adapters"
-            deploy-to:
-                description: "Choose whether to publish to test or prod"
-                type: string
-                default: "prod"
+                required: true
```

Comment:
We don't publish `dbt-adapters` internally, at least not yet, hence it should not be a default.